modules/observability/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-rhdh-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-admin-enabling-metrics-ocp-operator_{context}"]
+= Enabling metrics monitoring in a {product} Operator installation on an {ocp-short} cluster
+
+You can enable and view metrics for an Operator-installed {product} instance from the *Developer* perspective of the {ocp-short} web console.
+
+.Prerequisites
+
+* Your {ocp-short} cluster has link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[monitoring for user-defined projects] enabled.
+
+* You have installed {product} on {ocp-short} using the {product} Operator.
+
+* You have installed the {ocp-short} `oc` CLI.
+
+.Procedure
+
+Currently, the {product} Operator does not support creating a `ServiceMonitor` custom resource (CR) by default. You must complete the following steps to create a `ServiceMonitor` custom resource (CR) to scrape metrics from the endpoint.
+
+. Substitute the values for your environment and then run the following commands to create the `ServiceMonitor` CR:
++
+[source,terminal]
+----
+<custom_resource_name>=my-rhdh
+----
++
+[source,terminal]
+----
+<project_name>=my-project
+----
++
+[source,yaml]
+----
+cat <<EOF > /tmp/$<custom_resource_name>.ServiceMonitor.yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: <custom_resource_name>
+  namespace: <project_name>
+  labels:
+    app.kubernetes.io/instance: <custom_resource_name>
+    app.kubernetes.io/name: backstage
+spec:
+  namespaceSelector:
+    matchNames:
+      - <project_name>
+  selector:
+    matchLabels:
+      janus-idp.io/app: backstage-<custom_resource_name>
+  endpoints:
+  - port: http-backend
```

Comment:
```suggestion
  - port: backend
```
Same point for the port name, which is not correct: https://github.com/janus-idp/operator/blob/1.1.x/config/manager/default-config/deployment.yaml#L88 in 1.1 and https://github.com/janus-idp/operator/blob/1.2.x/config/manager/default-config/deployment.yaml#L97 in 1.2